acceptance/src/routes.js

```diff
@@ -1,6 +1,8 @@
 import AvatarPage from "./pages/AvatarPage";
+import MuiAvatarPage from "./pages/mui/AvatarPage";
```

Comment:
The imports are listed alphabetically, I think we should either keep that format or put all the `material-ui` imports together.